src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class FindCommand extends Command{
+    private final String fullCommand;
+
+    public FindCommand(String fullCommand) {
+        this.fullCommand = fullCommand;
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
+        String keyword = fullCommand.substring(4).strip();
+        taskList.displayFind(keyword);
+    }
+}
```

Comment:
Consider adding JavaDoc comments to your code to make people understand what the code is trying to do. If it is finding, then explain those with JavaDoc header.